Example/App.js

```diff
@@ -55,10 +60,23 @@ export default class App extends Component<{}, State> {
           accessTokenExpirationDate: authState.accessTokenExpirationDate,
           refreshToken: authState.refreshToken
         },
-        500
+        400
       );
     } catch (error) {
-      Alert.alert('Failed to log in', error.message);
+      console.log({ error })
+
+      if (error.code === 'RNAppAuth Error') {
+        const msg = error.message;
+        if (msg === 'User cancelled flow') {
```

Comment:
the same wont work on ios//iPhone. add "User cancelled." (including ".")  to make it work on ios//iPhone.
`if (msg === 'User cancelled flow' || msg === 'User cancelled.' )`

Reply:
```suggestion
        if (msg === 'User cancelled flow' || msg === 'User cancelled.') {
```